test/unit/specs/components/Thanks/ChallengeHeader.spec.js

```diff
@@ -49,7 +49,7 @@ describe('ChallengeHeader', () => {
 		const LoansFunded = getByText('1/2 loans funded');
 		expect(getByTestId('loans-funded')).toBe(LoansFunded);
 
-		const PercentageFunded = getByText('50% complete');
+		const PercentageFunded = getByText('0% complete');
```

Comment:
Shouldn't this be "50%" if 1/2 the funds are funded, like mentioned here on line 49?

Reply:
Oh right, this is the old component that is no longer active.